modules/libraries/standard/common/include/iot_taskpool.h

```diff
@@ -82,7 +82,7 @@
  *
  * This function should be called once by the application to initialize the one single instance of the system task pool.
  * An application should initialize the system task pool early in the boot sequence, before initializing any other library
- * that uses the system task pool. such as MQTT, Shadow, Defernder, etc.. An application should also initialize the system
+ * that uses the system task pool. such as MQTT, Shadow, Defender, etc.. An application should also initialize the system
```

Comment:
There are still typos in this line. Can we just say "... sequence, before initializing any other library. An application should also..."? Giving a partial list of example libraries is probably misleading.

Reply:
Fixed.